lib/identity_doc_auth/lexis_nexis/lexis_nexis_client.rb

```diff
@@ -34,7 +34,7 @@ def get_results(instance_id:)
         raise NotImplementedError
       end
 
-      def post_images(front_image:, back_image:, selfie_image:, liveness_checking_enabled: nil)
+      def post_images(front_image:, back_image:, selfie_image:, liveness_checking_enabled: nil, **)
```

Comment:
I'd make `cropping_mode` a default argument to match, otherwise we could end up with a typo that this `**` accepts that the acuant client would not

Reply:
The problem with this is that it's flagged as an unused variable. Although now that I look, we don't have Rubocop set up for this repository? But if we would, not sure how best to deal with that, aside from this or just explicitly disabling the rule for the occurrence.

Reply:
If Rubocop is enabled, I'd just override/disable it here as needed, Rubocop clearly isn't worrying about consistent method signatures across classes :]

Reply:
> I'd make `cropping_mode` a default argument to match, otherwise we could end up with a typo that this `**` accepts that the acuant client would not

Updated in 11293de.